mentat/code_context.py

```diff
@@ -86,7 +86,7 @@ def _get_files(
     files = {}
     for file in files_direct:
         if file.path not in excluded_files | excluded_files_from_dir:
-            files[file.path] = file
+            files[Path(os.path.realpath(file.path))] = file
```

Comment:
Well the real reason was because we were actually just putting a relative Path in there (when it was supposed to be an absolute path!); but yeah, the reason I use realpath instead of abspath is because of symlinks